Sources/TokamakCore/Views/TextField.swift

```diff
@@ -33,9 +34,21 @@ extension TextField where Label == Text {
     onCommit: @escaping () -> () = {}
   ) where S: StringProtocol {
     label = Text(title)
-    textBinding = text.projectedValue
+    textBinding = text
     self.onEditingChanged = onEditingChanged
     self.onCommit = onCommit
+    textFieldStyle = DefaultTextFieldStyle()
+  }
+
+  init(
+    _from textField: TextField,
```

Comment:
**nitpick:** Not sure if underscore here is supposed to indicate that something is internal?

```suggestion
    from textField: TextField,
```

Reply:
Yep! That's the idea since you're not supposed to be able to pass a TextFieldStyle directly into a TextField.

Reply:
The function is not `public` though, so nothing outside of `TokamakCore` can use it, which falls in the area of ["rule 1"](https://github.com/swiftwasm/Tokamak/pull/137/files#diff-04c6e90faac2675aa89e2176d2eec7d8R126) I attempted to describe in #137, but I'll let you review those rules first 🙂